src/Board.coffee

```diff
@@ -40,11 +40,11 @@ getRandomTile = ->
     pit: 35
     coin: 30
   insane =
-    dirt: 25
-    pit: 65
-    coin: 10
+    dirt: 50#25
```

Comment:
It was for before I decided to add for testing purposes the reveal all coins board method, and so I was literally trying to click all the coins to then see if the reveal all pits method worked. Clearly a wrong approach